File: Quivi.Pos.Api/Dtos/PosIntegration.cs
namespace Quivi.Pos.Api.Dtos
{
    public class PosIntegration
    {
        public required string Id { get; init; }
        public bool IsOnline { get; init; }
        public bool AllowsPayments { get; init; }
        public bool AllowsOpeningSessions { get; init; }
        public bool AllowsEscPosInvoices { get; init; }
        public bool AllowsAddingItemsToSession { get; init; }
        public bool AllowsRemovingItemsFromSession { get; init; }
    }
}

File: quivi.pos.react.app/src/hooks/api/Dtos/posintegrations/PosIntegration.ts
export interface PosIntegration {
    readonly id: string;
    readonly isOnline: boolean;
    readonly allowsPayments: boolean;
    readonly allowsOpeningSessions: boolean;
    readonly allowsEscPosInvoices: boolean;
    readonly allowsAddingItemsToSession: boolean;
    readonly allowsRemovingItemsFromSession: boolean;
    readonly allowsRefunds: boolean;
}